mava/wrappers/jumanji.py

```diff
@@ -117,3 +124,94 @@ def modify_timestep(self, timestep: TimeStep) -> TimeStep[Observation]:
 
         # Aggregate the list of individual rewards and use a single team_reward.
         return self.aggregate_rewards(timestep, modified_observation)
+
+
+class ConnectorWrapper(MultiAgentWrapper):
+    """Multi-agent wrapper for the MA Connector environment.
+
+    Do not use the AgentID wrapper with this env, it has implicit agent IDs.
+    """
+
+    def __init__(self, env: MaConnector, has_global_state: bool = False):
+        super().__init__(env)
+        self.has_global_state = has_global_state
+
+    def modify_timestep(
+        self, timestep: TimeStep
+    ) -> TimeStep[Union[Observation, ObservationGlobalState]]:
+        """Modify the timestep for the Connector environment."""
+
+        # TARGET = 3 = The number of different types of items on the grid.
+        def create_agents_view(grid: chex.Array) -> chex.Array:
+            positions = jnp.where(grid % TARGET == POSITION, True, False)
+            targets = jnp.where((grid % TARGET == 0) & (grid != EMPTY), True, False)
+            paths = jnp.where(grid % TARGET == PATH, True, False)
+            my_position = jnp.where(grid == POSITION, True, False)
+            my_target = jnp.where(grid == TARGET, True, False)
```

Comment:
The naming here is a bit confusing? Is this the position and target of a current agent?

Reply:
Yes, this is the per-agent position and target.

Reply:
Can we please name this in a way where this is easier to infer? Maybe something like `per_agent_position`, `per_agent_target`?

Reply:
Done, along with all the other changes.